### DISCUSSION TITLE
Coinbase not  showing in our project

### DISCUSSION THREAD

**Discussion by rahulnrt** [WalletConnect/walletconnect-monorepo#3890] [Developer Support] [answered]
When I put the Coinbase wallet id in my  project, while I connecting to Coinbase the i do not seen the Coinbase wallet in walletconnect modal

**Comment by glitch-txs** [comment, marked as answer]
Coinbase doesn't support WalletConnect protocol. You'll need to use the Web3Modal SDK with its connector

**Comment by rahulnrt** [comment]
Ok I am using web3modal sdk with coinbase connector it is work and connect to wallet but whenever i try to send transaction unable to send testnet transaction
